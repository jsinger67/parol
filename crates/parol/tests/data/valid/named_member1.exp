%start Start

%%

/* 0 */ Start: "a"@member1 : i32 "b"@member2 C member3;
/* 1 */ C: "c";
