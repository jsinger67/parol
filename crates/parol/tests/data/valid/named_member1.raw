%start Start

%%

Start: "a"@member1 : i32 "b"@member2 C member3;
C: "c";
