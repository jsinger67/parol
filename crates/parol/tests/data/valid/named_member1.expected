title: None
comment: None
start_symbol: Start
grammar_type: LLK
name: INITIAL;line_comments: [];block_comments: [];auto_newline_off: false;auto_ws_off: false;
Start: Alts(Alt(<0>T("a")@member1 : i32, <0>T("b")@member2, N(C)@member3));
C: Alts(Alt(<0>T("c")));
